O/openfhe_julia/build_tarballs.jl

```diff
@@ -78,7 +78,7 @@ products = [
 # Dependencies that must be installed before this package can be built
 dependencies = [
     BuildDependency(PackageSpec(;name="libjulia_jll", version=v"1.10.7")),
-    Dependency(PackageSpec(name="libcxxwrap_julia_jll", uuid="3eaa8342-bff7-56a5-9981-c04077f7cee7")),
+    Dependency(PackageSpec(name="libcxxwrap_julia_jll", uuid="3eaa8342-bff7-56a5-9981-c04077f7cee7", version=v"0.11.2")),
```

Comment:
Why not using compat instead? I always find it dodgy to specify the version for building abut not the compat: if the build version is important it's very likely compat would be very useful too

Reply:
Sorry, I didn't know how to specify the compat version essentially I want something like `"< 0.12“`. Would that be sufficient to set as version or is the correct keyword `compat`?

Reply:
Let's try it...